pif_ingestor/globus.py

```diff
@@ -0,0 +1,84 @@
+import os
+from globus_sdk import GlobusError, TransferData
+from mdf_forge import toolbox
+
+interval_time = 10
+default_collection = 35  # MDF Test collection
+
+def push_to_globus(paths, metadata={}, collection=default_collection, src_ep=None, verbose=False):
```

Comment:
what is `src_ep`?

Reply:
Source endpoint, counterpart to dest_ep. Possibly too short for its own good.